frontend/test/metabase/scenarios/models/reproductions/19737-data-picker-not-showing-moved-model.cy.spec.js

```diff
@@ -13,30 +13,76 @@ describe.skip("issue 19737", () => {
   it("should show moved model in the data picker without refreshing (metabase#19737)", () => {
     cy.visit("/collection/root");
 
-    openEllipsisMenuFor(modelName);
-    popover()
-      .contains("Move")
+    moveModel(modelName, "My personal collection");
+
+    cy.findByText("Moved model");
+
+    cy.findByText("New").click();
+    cy.findByText("Question")
+      .should("be.visible")
       .click();
 
-    modal().within(() => {
-      cy.findByText("My personal collection").click();
-      cy.findByText("Move").click();
-    });
+    cy.findByText("Models").click();
+
+    cy.findByText("Your personal collection").click();
+    cy.findByText(modelName);
+  });
+
+  it("should not show duplicate models in the data picker after it's moved from a custom collection without refreshing (metabase#19737)", () => {
```

Comment:
This is like the follow-up for the fix. After I changed `frontend/src/metabase/query_builder/components/saved-question-picker/SavedQuestionPicker.jsx` by adding `reload: true` the first test passed, but it could introduce other weird bugs. You could try reverting the change in `frontend/src/metabase/entities/schemas.js` to see the second test fail.